deploy.pl

```diff
@@ -36,15 +37,33 @@
 
 my $ua = Mojo::UserAgent->new;
```

Comment:
That's actually right. I think `$ua` is perfectly fine in Perl code, it is widely used acronym and Perl programmers know it.